### PR TITLE
[DOCS-6672] - Add autocomplete for APIs and script variables

### PR DESCRIPTION
**Pull request by oliveralfresco2018**

### CODE REVIEW

**Review by Artur-Piechuta — COMMENTED** [review]
Could you please add a screen shot below this text? (how autocomplete looks like)

**Comment by oliveralfresco2018** [comment]
image

**Review by Artur-Piechuta — APPROVED** [review]
Looks good to me

File: process-automation/latest/model/scripts.md [hunk 1]
```diff
@@ -40,7 +40,9 @@ To create a script:
 
 The Modeling Application contains two tabs for creating and managing scripts.
 
-The **Script Editor** is the GUI for modeling scripts by typing in the declared language. The **Metadata** contains the properties related to the script.
+The **Script Editor** is the GUI for modeling scripts by typing in the declared language. The editor has autocomplete functionality for APIs and script variables. The **Metadata** contains the properties related to the script.
+
+![auto complete]({% link process-automation/images/auto-complete.png %})
```
Comment (anxumalo) [inline process-automation/latest/model/scripts.md:45]: Maintenance consideration: we should standardize the term: autocomplete / auto complete / auto-complete even though they're not all visible (future consideration).

**Review by anxumalo — APPROVED** [review]
LGTM

**Pull request merged**